utils/netutils/netutils.go

```diff
@@ -213,6 +213,23 @@ func ParseTagRanges(ranges string, tagType string) ([]TagRange, error) {
 	return tagRanges, nil
 }
 
+// IsTagInRange verifies if the passed pktTag adheres to the valueRange
+func IsTagInRange(pktTag int, valueRangeStr string, encap string) bool {
+	// Parse the valueRange to form a []TagRange object
+	valueRanges, err := ParseTagRanges(valueRangeStr, "vlan")
+	if err != nil {
+		log.Errorf("Not able to pass valueRange. Err: %s", err)
+		return false
+	}
+
+	for _, valueRange := range valueRanges {
```

Comment:
@jainvipin not sure I get it. The ranges are created in CreateTenant and so it makes sense to call  ParseTagRange there. CreateNetwork needs to check if the provided vlan/vxlan fall into the configured range. As per your comment CreateNetwork should do ParseTagRanges(Range, encap, vlan/vxlan) - which should validate if PktTag is in range and return multi-values (range, bool)?